enterprise_access/settings/devstack.py

```diff
@@ -79,6 +79,7 @@
 ENTERPRISE_ADMIN_PORTAL_URL = 'http://localhost:1991'
 ENTERPRISE_CATALOG_URL = 'http://enterprise.catalog.app:18160'
 ENTERPRISE_SUBSIDY_URL = 'http://enterprise-subsidy.app:18280'
+ENTERPRISE_ACCESS_URL = 'http://enterprise-access.app:18270'
```

Comment:
Because this URL is being used in the API response for the client to use as a web request, `enterprise-access.app` won't work as that the internal devstack network url (i.e., for service-to-service communication). In this case, we need the URL to be `http://localhost:18270` for the MFE to be able to make the request.

Running this branch locally, and clicking "Enroll" button in learner portal to call the API at `http://enterprise-access.app:18270` throws an error.